cli/main.rs

```diff
@@ -1,5 +1,4 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
-#![deny(warnings)]
```

Comment:
Keep?

Reply:
I don't think that makes sense:
* This is the only place we explicitly add `[deny(warnings)]`. We don't even have this anywhere in CLI!
* On CI we run with `-Dwarnings` for all targets anyway.
* It gets in the way during development, because cargo will make a hard stop at the first pedantic nit.